input/templates/partials/catalog/pdp/reserve-in-store-btn.hbs

```diff
@@ -0,0 +1,4 @@
+<button type="submit" class="btn reserve-in-store-btn">
+  <img class="store-thumb" src="{{@root.meta.assetsPath}}img/store-icon-29-29px.png" alt="{{i18n "catalog:reserve.title"}}">
+  {{#if text}}<span class="hidden-xs">{{i18n "catalog:reserve.title"}}</span>{{/if}}
```

Comment:
In small devices I would show the text too. I think it would be quite difficult to understand the icon alone. What do you think?